PyTorchEx1.ipynb

```diff
@@ -0,0 +1,478 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    train_seq = torch.tensor(tokens_train['input_ids'])
</pre><p> </p><p>This goes cuda array interface rather than dlpack . Are there  performance implications of either  ? </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/rapids-examples/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>